tests/Makefile

```diff
@@ -90,7 +90,7 @@ ALB_DIR=$(pwd)
 ALB_TEST_DIR=tests
 
 # MIL locations
-MIL_DIR=$(HOME)/workspace/habit-lang/mil-tools
```

Comment:
This needs a better way..

We can have everyone having different relative paths of their mil-tools and alb compiler
Is there any generic way of solving this problem of relative paths?